.github/workflows/ci.yml

```diff
@@ -65,24 +65,24 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
+      - name: Checkout repository
```

Comment:
Can you revert these formatting changes? Orthogonal to this PR.

Reply:
Done